lib/hooks/gitlab/templates/pipeline.html.haml

```diff
@@ -0,0 +1,36 @@
+:ruby
+  status_name = -> (name) { name == 'success_with_warnings' ? 'success(warnings)' : name }
+  label_classes = {
+    'failed'  => 'label-danger',
+    'created' => 'label-info',
+    'manual'  => 'label-info',
+    'running' => 'label-info',
+    'success' => 'label-success',
+    'success(warnings)' => 'label-warning'
+  }
+  web_url = payload.project.web_url
+  p_status = status_name[payload.object_attributes.status]
+  p_label_class = label_classes[p_status].presence || 'label-default'
+  p_id = payload.object_attributes.id
+  p_url = "#{web_url}/pipelines/#{p_id}"
+
+%div.col-lg-12
```

Comment:
@masumiya Thanks for the fix! Unfortunately, it does not seem  to work very well with wider windows either :crying_cat_face:  Because it sounds a little difficult for me to deal with it using grid-layout, I think we should remove these `.col-*` classes in this template. What do you think?


![selection_306](https://user-images.githubusercontent.com/43346/53779949-8b11be80-3f45-11e9-8d1a-60b5cf3b2a42.png)

Reply:
I am sorry many times and many thanks. 
`.col- *` has been deleted.
Instead, I used a space to make a margin.

Reply:
Thanks :whale: